Automattic/wp-calypso
# Jetpack Cloud: remove tiers from regular plans

## Proposed Changes

* This PR removes tiered AI as an alias to the regular AI plans, since both could be virtually available at the same time (with the new AI plan not being tiered).

## Why are these changes being made?


* Theses changes are being made to support the new plans for Jetpack AI
* Context: p1HpG7-tT1-p2

## Testing Instructions



* If D159268-code hasn't been merged yet, follow the instructions there to get the backend set up
* Open the Jetpack Cloud live link and go to `/pricing`
* Confirm that you see the discounted price for AI and there's no mention of tiers when you click on "More about AI"

<img width="596" alt="image" src="https://github.com/user-attachments/assets/9ca7f0c2-2bcc-473b-a7a1-ac549bdbdea2">

* Also confirm that you see a disclaimer at the bottom of the page mentioning the high request capacity:

<img width="1092" alt="image" src="https://github.com/user-attachments/assets/bbc10fb0-43db-41cc-85a7-c3fc7ef73364">

* Click on "Proceed to checkout" and confirm the same on the checkout page

<img width="1041" alt="image" src="https://github.com/user-attachments/assets/e136b816-082e-405c-a98d-426bec806366">


## Pre-merge Checklist
